pkg/rancher-desktop/main/mainmenu.ts

```diff
@@ -179,13 +178,8 @@ function getWindowsApplicationMenu(): Array<MenuItem> {
 function getPreferencesMenuItem(): MenuItemConstructorOptions[] {
   return [
     {
-      label:               'Preferences',
-      visible:             true,
-      registerAccelerator: true,
-      accelerator:         'CmdOrCtrl+,',
-      click() {
-        openMain(true);
-      },
+      label:   'Preferences',
+      visible: true,
```

Comment:
This has the unfortunate tradeoff of losing the keyboard hints in the application menu. I'm wondering if we can retain the hints.

Reply:
I didn't realize this. I think retaining the shortcut info in the menu entries is important.